lib/membrane_rtc_engine/endpoints/webrtc_endpoint.ex

```diff
@@ -27,8 +27,6 @@ defmodule Membrane.RTC.Engine.Endpoint.WebRTC do
   alias Membrane.WebRTC
   alias Membrane.WebRTC.{EndpointBin, SDP}
 
-  # TODO: remove configuring via app env
```

Comment:
Sorry to tell you that, but this TODO is still valid, `Application.get_env` simply migrated to a different place.

Reply:
I know, I moved it to 204